### ISSUE TITLE
astral chain + windows/amd problems

### ISSUE THREAD

**Issue by soredake** [yuzu-emu/yuzu#6711]
When using opengl game uses all available vram after loading save and pc lags very bad (0-1fps in game), log:
[yuzu_log_opengl.txt](https://github.com/yuzu-emu/yuzu/files/6874426/yuzu_log_opengl.txt)
With vulkan, game is find after loading save, but crashes (after using all vram maybe) when using elevator, log:
[yuzu_log_vulkan.txt](https://github.com/yuzu-emu/yuzu/files/6874429/yuzu_log_vulkan.txt)

Windows 10 Pro 21H1, rx 560, high accuracy, garbage collection and astc acceleration enabled.

**Comment by FernandoS27** [comment]
I've heard of this elevator bug once. I was never ever able to reproduce it. Do you have a save.

**Comment by soredake** [comment]
Sure, here are my save.
[astral chain save.zip](https://github.com/yuzu-emu/yuzu/files/6874436/astral.chain.save.zip)

**Comment by v1993** [comment]
Just in case - does enabling garbage collection make any difference?

**Comment by soredake** [comment]
> 
> 
> Just in case - does enabling garbage collection make any difference?

Sadly no, i initially tested with it enabled, disabled does not make difference too.

**Comment by AnarkoDX** [comment]
yeah i have the same problem with the elevator in vulkan it crashes the emulator .opengl its unplayable 3-4 fps.

Ryzen 1600AF
Rx580 4GB (latest drivers)

**Comment by soredake** [comment]
Vulkan still crashes the emulator, and opengl is still 0-1 fps.

**Comment by soredake** [comment]
With new option for astc recompression set to BC3 game emulator no longer crashes.
